tools/authz_generate_rules_for_roles.py

```diff
@@ -0,0 +1,165 @@
+#! /usr/bin/env python3
+#
+# Copyright 2023 Flant JSC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This script is used for generating rules for user-authz roles to
+# ./modules/140-user-authz/docs/README.md and ./modules/140-user-authz/docs/README_RU.md.
```

Comment:
Looks great! Some additions:
1. it should be executed via go generate. We have a flow for code gen tool. First execute them on CI, second check that there is no diff.
2. 2. Why in Python?

Reply:
1. Oh, I got it, I'll rewrite
2. I choose python because it was faster for me to write. I'll rewrite it in go and include in CI